force-app/main/default/classes/SOQL.cls

```diff
@@ -143,15 +161,16 @@ public with sharing class SOQL {
      */
     public static List<String> getContactNamesByAccount(Id accountId) {
         // Create a list to hold the Contact LastNames
-
+        List<String> lastNames = new List<String>();
         // Query the Contacts
+        List<Contact> contacts = [SELECT Id, LastName FROM Contact WHERE AccountId = :accountId];
         // Loop through each Contact and add the LastName to the list
-        //for () {
-        //    
-        //}
+        for (Contact c:contacts) {
```

Comment:
It's okay for small blocks of code like this but try to use more descriptive names for variables instead of c